packages/lesswrong/server/ckEditor/postEditorConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { isMigrations } from "../../lib/executionEnvironment";
+
+export const getPostEditorConfig = () => {
```

Comment:
I'm slightly confused why this move was needed.

Reply:
`editorConfigs` is a js file that using `import` and `export`, so trying to import outside of the estrella build pipeline is non-trivial